### PR TITLE
change job handling

### PR DESCRIPTION
**Pull request by maxhansson**

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -203,6 +191,19 @@
     t.datetime "updated_at"
   end
 
+  create_table "csv_transactions", force: :cascade do |t|
```
Comment (qzio) [inline db/schema.rb]: What is this table used for?

**Comment by maxhansson** [comment]
It is wrong. The table shoun't be there

**Comment by qzio** [comment]
Then it should be removed from the Pull request.

**Comment by qzio** [comment]
:+1:
LGTM
r=@qzio

**Pull request merged**